apps/challenges/models.py

```diff
@@ -220,3 +220,15 @@ class ChallengeConfiguration(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_zip_configuration'
+
+
+class StarChallenge(TimeStampedModel):
```

Comment:
Umm...I don't think we really need this feature. In my opinion, we should be fine with this. @taranjeet please let me know if I am missing of some analytics that will be super useful for us with this feature.